rllib/agents/ppo/ppo_tf_policy.py

```diff
@@ -198,7 +198,8 @@ def postprocess_ppo_gae(
         # input_dict.
         if policy.config["_use_trajectory_view_api"]:
             # Create an input dict according to the Model's requirements.
-            input_dict = policy.model.get_input_dict(sample_batch, index=-1)
+            input_dict = policy.model.get_input_dict(
+                sample_batch, index="last")
```

Comment:
:+1: love it.

Reply:
Thinking ahead, is it possible we unify model.from_batch() and model.get_input_dict()? It seems currently only PPO is using this get_input_dict method so maybe we should remove it.

Thinking... `model.from_batch(sample_batch, batch_index="last")` would be pretty clean. This is probably out of scope of this PR though.